examples/gno.land/p/demo/users/users.gno

```diff
@@ -0,0 +1,31 @@
+package users
+
+import (
+	"std"
+	"strconv"
+)
+
+//----------------------------------------
+// Types
+
+type User struct {
+	Address std.Address
+	Name    string
+	Profile string
+	Number  int
+	Invites int
+	Inviter std.Address
+}
+
+func (u *User) String() string {
```

Comment:
with respect to my first comment, I think it makes sense to consistently place this in r/demo/users if we decide to keep rendering within realms.

also, I think that if we do keep it here, this should be `Render() string` (as it was originally). Another idea for a "rendering function" --  `Link() string` -- which returns `[@user](/r/demo/users:user)` -- this can be deferred to another PR.

Since `String() string` is mostly useful for debugging (as formatters understand String() very often), we could have `String()` return something like `@user (<address>)`.

Reply:
Done: c894748e83fd132a51eac1d3d7ba041b3badd36d